src/Event/DatasetIndexSubscriber.php

```diff
@@ -124,6 +130,28 @@ public function populateCalculatedFields(PostTransformEvent $event)
         }
     }
 
+    /**
+     * Forces array elements to float.
+     *
+     * @param array|float $coordinates
```

Comment:
Missing return annotation

Reply:
So, with the new PHP analysis style guides, this no longer needed. You either have a return type on the function, or the the docblock.